chromium_src/chrome/browser/certificate_manager_model.cc

```diff
@@ -69,7 +69,7 @@ net::NSSCertDatabase* GetNSSCertDatabaseForResourceContext(
 
 // static
 void CertificateManagerModel::Create(content::BrowserContext* browser_context,
-                                     const CreationCallback& callback) {
+                                     CreationCallback callback) {
```

Comment:
`callback`  should be `std::move`'d wherever it is passed, and also the invocation should be `std::move(callback).Run`

Reply:
whoops, late-night mishap.

also, what does this mean for `GetCertDBOnIOThread`? callback is used twice there it appears

Reply:
Okay a couple of things to get this correct,

1) Remove the copy of `CertificateManagerModel` from `chromium_src` and apply our changes as patch to upstream code, similar to how we do it for printing etc.

2) Make the OnceCallback change to CreationCallback in this patch.

3) Upstream part of that patch.